### PR TITLE
Initial (incomplete) texlive docs update for TL2023. [ARXIVCE-405] David

### PR DESCRIPTION
**Pull request by DavidLFielding**
This is a very rough draft to show how issues with TL2023 might be documented.

I still need to work on the markup and links.

### CODE REVIEW

**Review by jweiskoff — APPROVED** [review]
We should update this to say "every week". We're well into the tens of thousands per month.

File: source/help/faq/texlive.md [hunk 1]
```diff
@@ -20,8 +20,14 @@ cases you will want to add customized packages to your submission.
 Widely used packages may be added to our local tree for the convenience
 of large disciplines.
 
+We don't anticipate significant issues for articles developed under TeX
```
Comment (jimentwood) [inline source/help/faq/texlive.md:23]:
```suggestion
We do not anticipate significant issues for articles developed under TeX 

```

File: source/help/faq/texlive.md [hunk 2]
```diff
@@ -1,47 +1,113 @@
 # Notes about arXiv's TeXLive Version
 
-*Update 2020-10-01: arXiv updated to TeXLive 2020* arXiv is now running
-under [TeXLive 2020](http://tug.org/texlive), with a new, updated and
+### Update to TeX Live 2023
```
Comment (jimentwood) [inline source/help/faq/texlive.md:3]:
```suggestion
## Update to TeX Live 2023

```

File: source/help/faq/texlive.md [hunk 3]
```diff
@@ -1,47 +1,113 @@
...
-Authors have multiple options to resolve the encoding problem. Ideally,
-you will determine the correct encoding and add the following line to 
-your source:
+We don't anticipate significant issues for articles developed under TeX 
+Live 2023. If you are submitting articles developed under older TeX Live  
+releases you may experinece some of the issues below. We ran some tests 
+against TeX Live 2020 articles to get a sense for common problems 
+with older TeX Live releases.
+
+#### Common problems with articles prepared using older TeX Live releases
```
Comment (jimentwood) [inline source/help/faq/texlive.md:29]:
```suggestion
### Common problems with articles prepared using older TeX Live releases

```

File: source/help/faq/texlive.md [hunk 4]
```diff
@@ -1,47 +1,113 @@
...
+a single page (often due to date). Nearly 60% of older TL2020 article source 
+resulted in changes to multiple page of the resulting TL2023 PDF. 
+**If you are submitting source developed under an older TeX Live version
+it is very important to review the PDF generated by TL2023.**
+
+Below we detail some of the issues you may encounter.
+These may be of interest to authors who might be submitting updated 
+source for a new replacement version of an existing article.
+
+#### Package changes that cause compilation failures
```
Comment (jimentwood) [inline source/help/faq/texlive.md:47]:
```suggestion
### Package changes that cause compilation failures

```

File: source/help/faq/texlive.md [hunk 5]
```diff
@@ -1,47 +1,113 @@
...
+ 
+#####biblatex
```
Comment (jimentwood) [inline source/help/faq/texlive.md:49]:
```suggestion
####biblatex

```

File: source/help/faq/texlive.md [hunk 6]
```diff
@@ -1,47 +1,113 @@
...
+        ! Package biblatex Error: Patching \MakeUppercase failed.
+    Cause:
+        Including an old biblatex.sty file with your submission will cause this error.
+    Solution:
+        Remove old biblatex.sty file from your submission
+Also See:
+        [StackExchange](https://tex.stackexchange.com/questions/654476/fresh-texlive-2022-install-biblatex-patching-makeuppercase-failed-error)
+
+
+#####acmart class:
```
Comment (jimentwood) [inline source/help/faq/texlive.md:62]:
```suggestion
####acmart class:

```

File: source/help/faq/texlive.md [hunk 7]
```diff
@@ -1,47 +1,113 @@
...
 
-Please contact us if there are modules not include in TeX Live 2020 or
-our locally maintained tree that you would like to see added. In most
-cases you will want to add customized packages to your submission.
-Widely used packages may be added to our local tree for the convenience
-of large disciplines.
+See page 117 of [Glossaries User Manual](http://ctan.uni-altai.ru/macros/latex2e/contrib/glossaries/glossaries-user.pdf).
+See complete Change log at [Changes](http://ctan.math.utah.edu/ctan/tex-archive/macros/latex/contrib/glossaries/CHANGES)
+
+#####\thanks
```
Comment (jimentwood) [inline source/help/faq/texlive.md:102]:
```suggestion
#### \thanks

```

File: source/help/faq/texlive.md [hunk 8]
```diff
@@ -1,47 +1,113 @@
...
-    \usepackage[encoding name]{inputenc}
 
-Please review the `inputenc.sty` package's 
-[documentation](http://mirrors.ctan.org/macros/latex/base/inputenc.pdf) 
-to determine the correct options for your source. The second option is to 
-add `\UseRawInputEncoding` as the first line of
-your source file. Please see 
-[Tug News 28](https://www.latex-project.org/news/latex2e-news/ltnews28.pdf)
-for more details.
+#####Glossaries:
```
Comment (jimentwood) [inline source/help/faq/texlive.md:78]:
```suggestion
#### Glossaries: 

```

File: source/help/faq/texlive.md [hunk 9]
```diff
@@ -1,47 +1,113 @@
...
-to switch their development environments to the latest TeX Live 2020 (if
+Live 2020 will process fine under the new TeX tree we encourage authors
+to switch their development environments to the latest TeX Live 2023 (if
 you have not done so already).
 
-TeX Live 2020 expects LaTeX source files to be encoded in UTF-8 by default
-and there is no longer a forgiving fall through to UTF-8 encoding. Compilation
-without an encoding declaration and with characters outside of the UTF-8
-encoding will fail with an error.
+Please contact us if there are modules not include in TeX Live 2023 or
```
Comment (jimentwood) [inline source/help/faq/texlive.md:17]:
```suggestion
Please [contact us](https://arxiv.org/support) if there are modules not include in TeX Live 2023 or

```

File: source/help/faq/texlive.md [hunk 10]
```diff
@@ -94,7 +160,52 @@ TL2016-based tree.
...
+[Tug News 28](https://www.latex-project.org/news/latex2e-news/ltnews28.pdf)
+for more details.
+
+Addons: Many modules not included in the core TeX Live release and
+maintained in our local tree have been updated to the most recent
+version, so long as the update does not drastically change or break
+existing documents. Modules with little or no recent usage have been
+dropped from our local tree.
+
+Please contact us if there are modules not include in TeX Live 2020 or
```
Comment (jimentwood) [inline source/help/faq/texlive.md:202]:
```suggestion
Please [contact us](https://arxiv.org/support) if there are modules not include in TeX Live 2020 or

```

File: source/help/faq/texlive.md [hunk 11]
```diff
@@ -1,48 +1,124 @@
...
-your source file. Please see 
-[Tug News 28](https://www.latex-project.org/news/latex2e-news/ltnews28.pdf)
-for more details.
-
-Addons: Many modules not included in the core TeX Live release and
-maintained in our local tree have been updated to the most recent
-version, so long as the update does not drastically change or break
-existing documents. Modules with little or no recent usage have been
-dropped from our local tree.
+Please [contact us](https://arxiv.org/support) if there are modules not include in TeX Live 2023 or
```
Comment (jimentwood) [inline source/help/faq/texlive.md:18]: need to delete extra line

File: source/help/faq/texlive.md [hunk 12]
```diff
@@ -1,48 +1,124 @@
...
 
-Please contact us if there are modules not include in TeX Live 2020 or
 our locally maintained tree that you would like to see added. In most
 cases you will want to add customized packages to your submission.
 Widely used packages may be added to our local tree for the convenience
 of large disciplines.
 
+We do not anticipate significant issues for articles developed under TeX 
+Live 2023. If you are submitting articles developed under older TeX Live
```
Comment (jimentwood) [inline source/help/faq/texlive.md:25]:
```suggestion
Live 2023. If you are submitting articles developed under older TeX Live
```

**Review by jimentwood — APPROVED** [review]

File: source/help/faq/texlive.md (hunk 11 above)
Comment (DavidLFielding) [reply source/help/faq/texlive.md]: It's gone!

**Pull request merged**